### PR TITLE
[BACKLOG-36841], [BACKLOG-36721]-Accessibility-PUC Home Page.

### PR DESCRIPTION
**Pull request by phalguni-a**
@pentaho/wcag - Please review.

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 6m 1s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **333**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-834/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css [hunk 1]
```diff
@@ -8297,4 +8307,20 @@ body.bootstrap .modal.confirmationDialog .dialog div.body {
...
+}
+
+.custom-dropdown:hover .custom-dropdown-label,
+.custom-dropdown:focus-visible .custom-dropdown-label{
+  color: #FFFFFF;
+}
+
+.custom-dropdown-popup .gwt-MenuBar-vertical .gwt-MenuItem.gwt-MenuItem-selected{
+  background-color: #e0e0e0;
+  color: #000000;
```
Comment (dcleao) [inline widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css:8325]: I would ask you to first try to place these new styles instead `mantleRuby.css`, as that's where _most_ custom dropdown styles are placed, given these classes belong to a GWT component defined in mantle.

If for some reason, these need to be here, then, please place them near the existing styles (e.g. for `custom-dropdown-arrow`). AFAICT, these are placed here, and not in `mantleRuby.css`, for easy access to shared images.

File: widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css [hunk 2]
```diff
@@ -6838,6 +6842,11 @@ div.listbox .pentaho-listbox {
   border-color: #7C0B2B;
 }
 
+#getting-started .pentaho-button:focus-visible {
+    background-color: #7C0B2B;
+    border-color: #7C0B2B;
```
Comment (dcleao) [inline widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css:6847]: Indentation is wrong.

**Review by dcleao — CHANGES_REQUESTED** [review]
Apart from the comments, please fix the formatting. There are multiple cases such as the following where the last selector is not followed by a space between it and the opening brace.

```css
.card:focus-visible{
```

File: widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css (hunk 2 above)
Comment (phalguni-a) [reply widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css]: Did the required changes

File: widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css (hunk 1 above)
Comment (phalguni-a) [reply widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css]: Will be creating a PR for mantleRuby.css

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 6m 1s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **333**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-834/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by dcleao — APPROVED** [review]

**Review by hv-vamsi — APPROVED** [review]

**Pull request merged**